packages/authentication/test/unit/metadata-provider.ts

```diff
@@ -0,0 +1,74 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import {Context, Provider} from '@loopback/context';
+import {ParsedRequest} from '@loopback/core';
+import {AuthMetadataProvider, IMetadata, authenticate} from '../..';
```

Comment:
We should name the interface as `AuthMetadata`. I don't think we use `I` prefix for interfaces.